Dockerfile

```diff
@@ -1,14 +1,24 @@
-FROM openjdk:8u171-jdk-alpine3.7
+FROM debian:bullseye@sha256:a4c378901a2ba14fd331e96a49101556e91ed592d5fd68ba7405fdbf9b969e61
 
-ENV AGENT_VERSION=1.1.4 \
+ENV AGENT_VERSION=2.0.5 \
     JAVA_START_HEAP=32m \
     JAVA_MAX_HEAP=512m \
     LOG_LEVEL=INFO
 
 WORKDIR /app
 
-RUN apk add --no-cache shadow curl bash apache-ant \
-    && curl -LO https://github.com/awslabs/amazon-kinesis-agent/archive/${AGENT_VERSION}.tar.gz \
+# Setup and install Java 1.8
```

Comment:
I can't see a reference to the specific version of Java being installed but I trust you :)

Reply:
ah yah it's a prereq to the line before where you run `openjdk-8-jdk`